RiotSharp/Endpoints/TftSummonerEndpoint/TftSummoner.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Newtonsoft.Json;
+using RiotSharp.Misc.Converters;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RiotSharp.Endpoints.TftSummonerEndpoint
+{
+    public class TftSummoner : TftSummonerBase
+    {
```

Comment:
It might be better to merge this with `TftSummonerBase`, because `TftSummonerBase` isn't used anywhere else.